doc/classes/Object.xml

```diff
@@ -220,7 +220,7 @@
 			<return type="bool" />
 			<param index="0" name="property" type="StringName" />
 			<description>
-				Override this method to customize the given [param property]'s revert behavior. Should return [code]true[/code] if the [param property] can be reverted in the Inspector dock. Use [method _property_get_revert] to specify the [param property]'s default value.
+				Override this method to customize the given [param property]'s revert behavior. Should return [code]true[/code] if the [param property] has a custom default value and is revertable in the Inspector dock. Use [method _property_get_revert] to specify the [param property]'s default value.
```

Comment:
```suggestion
				Override this method to customize the given [param property]'s revert behavior. Should return [code]true[/code] if the [param property] has a custom default value and is revertible in the Inspector dock. Use [method _property_get_revert] to specify the [param property]'s default value.
```
or
```suggestion
				Override this method to customize the given [param property]'s revert behavior. Should return [code]true[/code] if the [param property] has a custom default value and can be reverted in the Inspector dock. Use [method _property_get_revert] to specify the [param property]'s default value.
```

Reply:
I wanted to convey that the property can be reverted, but returning `true` does not force it (i.e. default is still checked).